simulation/main.go

```diff
@@ -48,12 +48,11 @@ const (
 )
 
 var (
-	nodebug    = flag.Bool("nodebug", false, "disable debug logging")
-	count      = flag.Int("count", 7, "node count")
-	blocked    = flag.Int("blocked", -1, "blocked validator (payloads from him/her are dropped)")
-	txPerBlock = flag.Int("txblock", 1, "transactions per block")
```

Comment:
Let's not remove this from parameters, but instead use directly in `GetVerified`.